### PR TITLE
KEYCLOAK-17491 Move the key settings to the new Keys tab

### PR DESCRIPTION
**Pull request by y-tabata**
Code PR: https://github.com/keycloak/keycloak/pull/7874

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
@y-tabata Thanks, These changes look good to me. However the PR needs to wait to be merged with the related codebase PR https://github.com/keycloak/keycloak/pull/7874 .

Also rebase is needed. Maybe you can just remove the file "client-scopes-evaluate.png" from this PR? If the only reason to include this file is the new "Keys" tab, then it is completely fine to remove the file "client-scopes-evaluate.png" from this PR (The tabs at the top do not need to be 100% accurate in the docs screenshots as long as the "main content" itself as accurate.

@andymunro Do you have a chance to review this please?

File: server_admin/topics/clients/client-oidc.adoc [hunk 1]
```diff
@@ -184,8 +184,7 @@ In Key Encryption, the client generates a key pair of asymmetric cryptography. T
 
 The client needs to pass their public key for encrypting CEK onto {project_name}. {project_name} supports downloading public keys from the URL the client provides. The client needs to provide their public keys according to https://tools.ietf.org/html/rfc7517[Json Web Keys (JWK)] specification. The way to do so is defined in `Signed JWT` of <<_client-credentials, Confidential Client Credentials>>. The detailed procedure is as follows:
 
-* open the client's `Credentials` tab
-* select `Signed Jwt` from `Client Authenticator` pulldown menu
+* open the client's `Keys` tab
 * set ON to `JWKS URL` switch
```
Comment (andymunro) [inline server_admin/topics/clients/client-oidc.adoc]: toggle JWKS URL to ON

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good.  Just one change.

File: server_admin/topics/clients/client-oidc.adoc (hunk 1 above)
Comment (y-tabata) [reply server_admin/topics/clients/client-oidc.adoc]: @andymunro Changed.

**Comment by y-tabata** [comment]
@mposolda @andymunro Thanks for reviewing. I rebased and changed according to your comment.

**Review by mposolda — APPROVED** [review]
@y-tabata @andymunro Thanks

**Pull request merged**